src/components/CohortBuilder/Categories/index.js

```diff
@@ -128,17 +137,36 @@ class Categories extends React.Component {
     this.props.onSqonUpdate(...args);
   }
 
+  // searching should not open quick filters
   handleSearchField(fieldName) {
-    this.setState({ currentSearchField: fieldName });
+    let currentCategory = null;
+    if (CATEGORY_FIELDS.clinical.indexOf(fieldName) > -1) {
```

Comment:
`CATEGORY_FIELDS.clinical.includes(fieldName)` ? :)

Reply:
A more maintainable approach:

```
const excludedCategories = ['searchAll', 'quickSearch'];
const currentCategoryKey = Object.keys(CATEGORY_FIELDS)
  .filter(key => !excludedCategories.includes(key))
  .find(key => CATEGORY_FIELDS[key].includes(fieldName));
const currentCategory = CATEGORY_FIELDS[currentCategoryKey];
```

Also, `['searchAll', 'quickSearch']` couyld be stored in `src/components/CohortBuilder/Categories/index.js`.

Reply:
much cleaner thank you :)